locksmith/src/controllers/hookController.ts

```diff
@@ -53,15 +53,24 @@ export async function subscribe(
     return createdHook
   }
 
-  // If changing mode, reconfirm the intent and update
-  if (hook?.mode !== hub.mode) {
+  // If changing mode or lease, reconfirm the intent and update
+  if (hook?.mode !== hub.mode || hub.lease_seconds) {
     hook.mode = hub.mode
+
+    const expiration = hub.lease_seconds
+      ? getExpiration(hub.lease_seconds)
```

Comment:
that would cause the expiration to change every time someone makes any change though. Should we allow changing expiration on mode to "unsubscribe" too?